### PR TITLE
[DX-1041] Sync 1.4.3 release notes

### PR DESCRIPTION
**Pull request by carolwkc**
### For internal users - Please add a Jira DX PR ticket to the subject!
<br>

Release notes for Sync 1.4.3

---

### Preview Link

[preview](https://deploy-preview-4082--tyk-docs.netlify.app/docs/nightly/product-stack/tyk-sync/release-notes/sync-1.4/)

### Description

<br>

#### Screenshots (if appropriate)
<br>

## Checklist

- [ ] I have added a **preview link** to the PR description.
- [ ] I have [reviewed the guidelines](https://github.com/TykTechnologies/tyk-docs/blob/master/CONTRIBUTING.md) for contributing to this repository.
- [ ] I have [read the technical guidelines](https://github.com/TykTechnologies/tyk-docs/blob/master/CONTRIBUTING-TECHNICAL-GUIDE.md) for contributing to this repository.
- [ ] Make sure you have started *your change* off *our latest `master`*.
- [ ] I **labelled** the PR

___

## **Type**
enhancement, documentation

___

## **Description**
- Introduced Tyk Sync 1.4.3 release notes with no breaking changes or deprecations.
- Provided upgrade instructions for users on version 1.4.x or older.
- Highlighted the release's compatibility enhancements and the update to Golang 1.21.
- Addressed security concerns by resolving specific CVEs.

___

## **Changes walkthrough**
<table><thead><tr><th></th><th align="left">Relevant files</th></tr></thead><tbody><tr><td><strong>Documentation</strong></td><td><table>
<tr>
  <td>
    <details>
      <summary><strong>sync-1.4.md</strong><dd><code>Tyk Sync 1.4.3 Release Notes Update</code>&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; </dd></summary>
<hr>

tyk-docs/content/product-stack/tyk-sync/release-notes/sync-1.4.md
<li>Added 1.4.3 release notes for Tyk Sync.<br> <li> Detailed upgrade instructions, release highlights, and download links.<br> <li> Updated API definitions compatibility and Golang version used.<br> <li> Listed resolved CVEs for security enhancements.

</details>
    

  </td>
  <td><a href="https://github.com/TykTechnologies/tyk-docs/pull/4082/files#diff-3bbcf95542e7da6f0faf921979a2fe54b5e2a3b8851764d4dbfd9e9993a9419f">+55/-0</a>&nbsp; &nbsp; </td>
</tr>                    
</table></td></tr></tr></tbody></table>

___

> ✨ **PR-Agent usage**:
>Comment `/help` on the PR to get a list of all available PR-Agent tools and their descriptions

### CODE REVIEW

File: tyk-docs/content/product-stack/tyk-sync/release-notes/sync-1.4.md [hunk 1]
```diff
@@ -10,6 +10,50 @@ description: "Release notes documenting updates, enhancements, fixes and changes
...
+There are no deprecations in this release.
+
+#### Upgrade instructions
+If you are using a 1.4.x version, we advise you to upgrade ASAP to this latest release. If you are on an older version, you should skip 1.4.0 and upgrade directly to this release.
+
+#### Release Highlights
+This release enhances compatibility as detailed in the [changelog]({{< ref "#Changelog-v1.4.3">}}) below.
+
+#### Downloads
+- [Docker image to pull](https://hub.docker.com/layers/tykio/tyk-sync/v1.4.3/images/TBC?context=explore)
```
Comment (dcs3spp) [inline tyk-docs/content/product-stack/tyk-sync/release-notes/sync-1.4.md]: Remember to add docker image and release date

File: tyk-docs/content/product-stack/tyk-sync/release-notes/sync-1.4.md [hunk 2]
```diff
@@ -10,6 +10,50 @@ description: "Release notes documenting updates, enhancements, fixes and changes
...
+#### Changelog {#Changelog-v1.4.3}
+
+##### Updated
+
+<ul>
+<li>
+<details>
+<summary>API definitions supported up to Tyk Gateway v5.3.0 </summary>
+
+Tyk Sync supports Tyk API definitions up to Tyk Gateway v5.3.0. Please use this version with Tyk Gateway v5.3.0+.
```
Comment (dcs3spp) [inline tyk-docs/content/product-stack/tyk-sync/release-notes/sync-1.4.md]: Do you mean use Tyk Sync 2.4.3+ for v5.3.0?

**Review by dcs3spp — CHANGES_REQUESTED** [review]
@caroltyk PR LGTM. Remember to add download for docker image when available.

Are there any dependencies and compatibilities that can be included for Tyk Sync?

Marking as draft until release date and docker image available

File: tyk-docs/content/product-stack/tyk-sync/release-notes/sync-1.4.md (hunk 2 above)
Comment (carolwkc) [reply tyk-docs/content/product-stack/tyk-sync/release-notes/sync-1.4.md]: Tyk Sync 1.4.3+ will work with Tyk 5.3.0

Comment (dcs3spp) [reply tyk-docs/content/product-stack/tyk-sync/release-notes/sync-1.4.md]: Thanks @caroltyk 

Can we say:

```suggestion
Tyk Sync v1.4.3 offers backward compatibility with Tyk API definitions for Gateway versions prior to v5.3.0. Please use Tyk Sync v1.4.3+ for compatibility with Tyk API definitions for Tyk Gateway v5.3.0+.
```

File: tyk-docs/content/product-stack/tyk-sync/release-notes/sync-1.4.md [hunk 3]
```diff
@@ -10,6 +10,52 @@ description: "Release notes documenting updates, enhancements, fixes and changes
...
+##### Updated
+
+<ul>
+<li>
+<details>
+<summary>API definitions supported up to Tyk Gateway v5.3.0 </summary>
+
+Tyk Sync v1.4.3 offers backward compatibility with Tyk API definitions for Gateway versions prior to v5.3.0. Please use Tyk Sync v1.4.3+ for compatibility with Tyk API definitions for Tyk Gateway v5.3.0+. 
+
+Tyk Sync is currently using the classic Dashboard API endpoints to migrate APIs. OAS API Category is not supported by Tyk Sync yet. Also, the use of [`--allow-unsafe-oas`]({{<ref "/tyk-sync">}}) flag is required for Tyk Sync to work with OAS API.
```
Comment (komalsukhani) [inline tyk-docs/content/product-stack/tyk-sync/release-notes/sync-1.4.md]: Users needs to set `allow-unsafe-oas` in Dashboard configuration.

**Review by komalsukhani — CHANGES_REQUESTED** [review]

**Comment by komalsukhani** [comment]
@caroltyk We also fixed CVEs mentioned in this ticket https://tyktech.atlassian.net/browse/TT-11230

File: tyk-docs/content/product-stack/tyk-sync/release-notes/sync-1.4.md (hunk 3 above)
Comment (carolwkc) [reply tyk-docs/content/product-stack/tyk-sync/release-notes/sync-1.4.md]:
```suggestion
Tyk Sync is currently using the classic Dashboard API endpoints to migrate APIs. OAS API Category is not supported by Tyk Sync yet. Also, Dashboard configuration [allow-unsafe-oas]({{<ref "/tyk-dashboard/configuration#allow_unsafe_oas">}}) and the use of [`--allow-unsafe-oas`]({{<ref "/tyk-sync">}}) flag in Tyk Sync is required for Tyk Sync to work with OAS API.
```

**Comment by carolwkc** [comment]
> @caroltyk We also fixed CVEs mentioned in this ticket https://tyktech.atlassian.net/browse/TT-11230

@komalsukhani  
CVE added

**Comment by dcs3spp** [comment]
@andyo-tyk To facilitate reviewing the PR queue we are closing some PRs that are in draft status, e.g. awaiting docker images, videos, images etc. You can still update the PR and then open it again when ready for review.

cc: @letzya

File: tyk-docs/content/product-stack/tyk-sync/release-notes/sync-1.4.md [hunk 4]
```diff
@@ -10,6 +10,60 @@ description: "Release notes documenting updates, enhancements, fixes and changes
...
+##### Updated
+
+<ul>
+<li>
+<details>
+<summary>API definitions supported up to Tyk Gateway v5.3.0 </summary>
+
+Tyk Sync v1.4.3 offers backward compatibility with Tyk API definitions for Gateway versions prior to v5.3.0. Please use Tyk Sync v1.4.3+ for compatibility with Tyk API definitions for Tyk Gateway v5.3.0+. 
+
+Tyk Sync is currently using the classic Dashboard API endpoints to migrate APIs. OAS API Category is not supported by Tyk Sync yet. Also, Dashboard configuration [allow-unsafe-oas]({{<ref "/tyk-dashboard/configuration#allow_unsafe_oas">}}) and the use of [--allow-unsafe-oas]({{<ref "/tyk-sync">}}) flag in Tyk Sync is required for Tyk Sync to work with OAS API.
```
Comment (komalsukhani) [inline tyk-docs/content/product-stack/tyk-sync/release-notes/sync-1.4.md]:
```suggestion
Tyk Sync is currently using the classic Dashboard API endpoints to migrate OAS APIs. OAS API Category is not supported by Tyk Sync yet. Also, Dashboard configuration [allow-unsafe-oas]({{<ref "/tyk-dashboard/configuration#allow_unsafe_oas">}}) and the use of [--allow-unsafe-oas]({{<ref "/tyk-sync">}}) flag in Tyk Sync is required for Tyk Sync to work with OAS API.
```

**Review by komalsukhani — APPROVED** [review]

**Comment by letzya** [comment]
@caroltyk Your PR has been inactive without updates or interactions since Feb 2024. As part of our ongoing housekeeping process to improve our review efficiency and maintain a focused queue, we have closed it. Please note that this action is purely procedural, and you are welcome to reopen your PR at any time when ready to proceed or require further review

File: tyk-docs/content/product-stack/tyk-sync/release-notes/sync-1.4.md [hunk 5]
```diff
@@ -10,6 +10,61 @@ description: "Release notes documenting updates, enhancements, fixes and changes
...
+<ul>
+<li>
+<details>
+<summary>Tyk Sync updated to use [Golang 1.21](https://tip.golang.org/doc/go1.21) </summary>
+
+Tyk Sync is using Golang 1.21 Programming Language starting with the 1.4.3 release. This brings improvements to the code base and allows us to benefit from the latest features and security enhancements in Go. 
+</details>
+</li>
+</ul>
+
```
Comment (letzya) [inline tyk-docs/content/product-stack/tyk-sync/release-notes/sync-1.4.md:69]: The sentence about category is lost in the crowd and doesn't explain what it actually means.

Comment (carolwkc) [reply tyk-docs/content/product-stack/tyk-sync/release-notes/sync-1.4.md:69]: @letzya updated wordings there

**Review by letzya — APPROVED** [review]

**Review by dcs3spp — APPROVED** [review]

**Pull request merged**